src/Core/Common/GetTypeBasicInformation/Code.py

```diff
@@ -20,7 +20,7 @@
 
 print("Min: " + str(itk.NumericTraits[MyType].min()))
 print("Max: " + str(itk.NumericTraits[MyType].max()))
-print("Zero: " + str(itk.NumericTraits[MyType].Zero()))
+print("Zero: " + str(itk.NumericTraits[MyType].Zero))
```

Comment:
This is still failing I think. Not sure why `itk.NumericTraits` does not have a `Zero` attribute.

Reply:
We need to use `.ZeroValue()` -- we can inspect the functions available with `dir(itk.NumericTraits[MyType])`

Reply:
:+1: Removed the call in 63d0bf6: a call to print the `ZeroValue()` exists immediately after.